functional/vfio/run.sh

```diff
@@ -107,19 +118,40 @@ setup_configuration_file() {
 		fi
 	done
 
+	if [ "$hypervisor" = "cloud-hypervisor" ]; then
+		CONFIG_FILE="${tmp_data_dir}/${hypervisor_clh}"
+		cp "${config_options_path}/${hypervisor_clh}" "${CONFIG_FILE}"
+	elif [ "$hypervisor" = "qemu" ]; then
+		CONFIG_FILE="${tmp_data_dir}/${hypervisor_qemu}"
+		cp "${config_options_path}/${hypervisor_qemu}" "${CONFIG_FILE}"
+	fi
```

Comment:
what happen if the file not exist?. I think  this `if..else` should be part of the previous `for`

```sh
for file in $(kata-runtime --kata-show-default-config-paths); do
	confif_dir=$(dirname ${file})
	if [ "$hypervisor" = "cloud-hypervisor" ]; then
		config_filename=${clh_config_file}"
	elif [ "$hypervisor" = "qemu" ]; then
		config_filename=${qemu_config_file}"
	fi
	config_file=${config_dir}/${config_filename}
	if [ -f "${config_file}" ]; then
		cp -a "${config_file}" "${CONFIG_FILE}"
		break
	fi
done

```

Reply:
Yep this will work but I'm confused about one thing though. Running `kata-runtime --kata-show-default-config-paths`  shows `/etc/kata-containers/configuration.toml` and `/usr/share/defaults/kata-containers/configuration.toml`, although the first one doesn't exist (the directory doesn't exist). Is this expected behavior?

Reply:
yes, this is expected, like CL, configuration files under `/etc` take precedence (stateless)

Reply:
@devimc I applied these changes, thanks for the feedback